### ISSUE TITLE
Incorrect Svelte Syntax Highlighting 

### ISSUE THREAD

**Issue by hyunbinseo** [github-linguist/linguist#6673]
## Describe the bug

Svelte's `$: if( ) { }` syntax seems to be broken. [GitHub Gist](https://gist.github.com/hyunbinseo/2b5944071cd8d37e8b0a7ac0fa2e4b2a)

Currently, `lang="ts"` is required to correctly highlight this syntax.

![image](https://github.com/github-linguist/linguist/assets/47051820/151a9a0c-51a0-46ad-bf04-6a29bf6369ff)

### Expected behaviour

In the above example, the highlight should be identical for JS and TS.

**Comment by lildude** [comment]
Closing as duplicate of pinned issue https://github.com/github-linguist/linguist/issues/6668

**Comment by lildude** [comment]
Actually, cancel that. This will be an issue with the upstream grammar. Please open an issue there.

**Comment by hyunbinseo** [comment]
Thanks for the comments. Am I supposed to create an issue on [sveltejs/language-tools](https://github.com/sveltejs/language-tools)?

The syntax highlighting works on all other platforms - it is just broken on [github.com](https://gist.github.com/hyunbinseo/2b5944071cd8d37e8b0a7ac0fa2e4b2a).

This is the reason I've created this issue in this repository.

**Comment by lildude** [comment]
Yes. We take the grammar as provided by that repo so if it's not highlighting correctly, it something in the grammar... except if it's caused by the issue I originally highlighted.
